src/System.Runtime.Extensions/tests/System/BitConverterArray.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public class BitConverterArray : BitConverterBase
+    {
+        public override void ConvertFromBool(bool boolean, int expected)
+        {
+            Assert.Equal(expected, BitConverter.GetBytes(boolean)[0]);
+        }
+
+        public override void ConvertFromShort(short num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromChar(char character, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(character));
+        }
+
+        public override void ConvertFromInt(int num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromLong(long num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromUShort(ushort num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromUInt(uint num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromULong(ulong num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromFloat(float num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ConvertFromDouble(double num, byte[] byteArr)
+        {
+            Assert.Equal(byteArr, BitConverter.GetBytes(num));
+        }
+
+        public override void ToChar(int index, char expected)
+        {
+            Assert.Equal(expected, BitConverter.ToChar(s_toCharByteArray, index));
```

Comment:
It'd be better to pass all of the inputs into the function as args, rather than pulling one from a static.  That way, the test is only relying on the data explicitly given to it, and the caller has to provide the inputs and the expected outputs together... makes it less likely we'll have mistakes.